packages/smart-contracts/src/lib/artifacts/Erc20SwapConversion/index.ts

```diff
@@ -18,6 +18,10 @@ export const erc20SwapConversionArtifact = new ContractArtifact<ERC20SwapToConve
           address: '0x1d6B06C6f7adFd9314BD4C58a6D306261113a1D4',
           creationBlockNumber: 9447192,
         },
+        goerli: {
```

Comment:
You can keep the 2nd one only. Cf. the end of the file, the default version is `0.2.0`